types/ckeditor__ckeditor5-engine/src/view/element.d.ts

```diff
@@ -44,4 +48,5 @@ export default abstract class Element extends Node {
     hasClass(className: string): boolean;
     hasStyle(property: string): boolean;
     isSimilar(otherElement: Element): boolean;
+    _appendChild(item: Item): void;
```

Comment:
Should this be `private`?

Reply:
Yes, and so it doesn't need to be here. I'm not really sure how this ended up here, I'll remove it.